src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -8181,7 +8181,7 @@ namespace InternalHeatGains {
                     state.dataHeatBal->ZoneRpt(it->first).ITEAdjReturnTemp = state.dataHeatBal->Zone(it->first).AdjustedReturnTempByITE;
                 }
             }
-            it++;
+            ++it;
```

Comment:
[src/EnergyPlus/InternalHeatGains.cc:8184]:(performance),[postfixOperator],Prefer prefix ++/-- operators for non-primitive types.